projects/arcanum/index.js

```diff
@@ -10,11 +10,18 @@ const ASSETS_CONTRACTS = [
     '0x539bde0d7dbd336b79148aa742883198bbf60342',
 ];
 
+const SPI_CONTRACT = '0xbB5b3D9F6B57077b4545ea9879ee7fD0BDB08dB0';
+const SPI_ASSETS_CONTRACTS = [
+    ADDRESSES.arbitrum.USDT,
+    ADDRESSES.arbitrum.WBTC,
+    ADDRESSES.arbitrum.WSTETH,
+];
+
 
 module.exports = {
-    methodology: 'counts the quantities of all tokens in multipool contracts.',
+    methodology: 'counts the quantities of all tokens in all multipool contracts.',
     start: 1000235,
     arbitrum: {
-        tvl: sumTokensExport({ owner: ARBI_CONTRACT, tokens: ASSETS_CONTRACTS }),
+        tvl: sumTokensExport({ owners: [ARBI_CONTRACT, SPI_CONTRACT], tokens: ARBI_ASSETS_CONTRACTS.concat(SPI_ASSETS_CONTRACTS) })
```

Comment:
can you change the line to below? it is bit more efficient

    tvl: sumTokensExport({ ownerTokens: [[ARBI_ASSETS_CONTRACTS, ARBI_CONTRACT], [SPI_ASSETS_CONTRACTS, SPI_CONTRACT]], })

Reply:
> can you change the line to below? it is bit more efficient
> 
> ```
> tvl: sumTokensExport({ ownerTokens: [[ARBI_ASSETS_CONTRACTS, ARBI_CONTRACT], [SPI_ASSETS_CONTRACTS, SPI_CONTRACT]], })
> ```

done